### ISSUE TITLE
error preventing app from startup : wrong imports

### ISSUE THREAD

**Issue by iameru** [devhausleipzig/camp-8-midterm-group-2#71]
In the current main there was merges which introduced a bug. the site can not load anymore. 
in the console the error is shown: 
`Uncaught SyntaxError: ambiguous indirect export: default main.tsx:27:7`

there on line 27 in `src/main.tsx` there is an import which can not work as the Navbar component has no default export. 

with `git blame src/main.tsx` i found the commit c003f29c for line 27. My knowledge of tracing git sadly ends here.

**Comment by iameru** [comment]
could be the solution
```
diff --git a/src/main.tsx b/src/main.tsx
index 2acf70f..6a4ef31 100644
--- a/src/main.tsx
+++ b/src/main.tsx
@@ -24,7 +24,7 @@ import {
 } from "./routes";
 import { BookingLayout } from "./layouts/BookingLayout";
 import { Movies, MoviesLoader } from "./routes/Movies";
-import NavBar from "./layouts/NavBarLayout";
+import { NavBarLayout } from "./layouts/NavBarLayout";
 
 const movieDetailChildren = [
   {
@@ -64,7 +64,7 @@ const router = createBrowserRouter([
   },
   {
     path: "/",
-    element: <NavBar />,
+    element: <NavBarLayout />,
     children: [
       {
         index: true,
```

**Comment by amyria3** [comment]
i made a patch for NavBar default export :)

**Comment by Sebi900** [comment]
i guess you found the bug, that should be erverthing wrong about it @iameru

**Comment by iameru** [comment]
patch was apparently merged in main with  #72
That was the fix, closing.
